guides/hack/02-source-code-fundamentals/37-namespaces.md

```diff
@@ -52,11 +58,16 @@ namespace NS3\Sub1;
 ...                   // __NAMESPACE__ is "NS3\Sub1"
 }
 ```
+## Importing from other Namespaces
+With the `use` keyword, a namespace can import one or more member names into a scope, optionally giving them each an alias.
 
-In this case, the namespace extends from the opening brace to the closing brace.
+When importing many names, use `{ ... }`.
+
+```Hack
+use NS1\{C, I, T}; // instead of `NS1\C, NS1\I, NS1\T`
```

Comment:
Please prefer `use type` or `use namespace`. This is generally preferred to avoid ambiguous situations. Hhast encourages the `use [kind]` style too. https://github.com/hhvm/hhast/blob/main/src/Linters/UseStatementWIthoutKindLinter.hack

Reply:
I learned something today too :-)